### ISSUE TITLE
Home needs a H1

### ISSUE THREAD

**Issue by pdpinch** [mitodl/ocw-hugo-themes#89]
For SEO and for accessibility, [the first heading on a page should be a H1](https://www.w3.org/WAI/tutorials/page-structure/headings/).

On the home page, it can go here:

https://github.com/mitodl/ocw-hugo-themes/blob/44cc97574daa3dce714e47503bb42177ee0917a0/www/layouts/home.html#L54-L58
